### ISSUE TITLE
Improvments to "all drinks" view

### ISSUE THREAD

**Issue by tistatos** [klaseskilson/vattentornet#83]
This view should:
- Be sortable
- look prettier (background is too short for content)
- divide drinks into two categories, (in-stock, out of stock)

**Comment by klaseskilson** [comment]
@tistatos There is no need to close issues manually, if referenced correctly they will close automatically once the referencing commit is merged into `master`.

**Comment by tistatos** [comment]
aah I'm always confused with when it actually closes

**Comment by klaseskilson** [comment]
Yeah, it is a bit unclear sometimes. Doesn't really fit perfectly into the git flow workflow.

There is a little hint though:
<img width="720" alt="screen shot 2016-01-28 at 12 13 38" src="https://cloud.githubusercontent.com/assets/1674076/12642864/c0f86c7e-c5b8-11e5-8b86-c9f54f90c7fb.png">
